src/boundingmesh/CMakeLists.txt

```diff
@@ -4,9 +4,16 @@ find_package(Eigen 3 REQUIRED)
 find_package(Coin)
```

Comment:
Because this a quite a change, run all tests with docker.
Please install docker and follow the command documented in https://github.com/gaschler/bounding-mesh#contribute